apps/meteor/server/services/messages/hooks/BeforeSaveJumpToMessage.ts

```diff
@@ -71,6 +71,10 @@ export class BeforeSaveJumpToMessage {
 			useRealName: boolean;
 		};
 	}): Promise<IMessage> {
+		if (message?.attachments) {
```

Comment:
I'd say we should verify if this is an edited message here since this case only applies to edited messages.

Reply:
I did that change and got a test unit error. It happened because we were expecting this function to also not duplicate quotes from message attachments that have already been provided (on the first time a message is sent, not only on edit)

Also, I'm thinking we should **always** remove existing quote attachments and rebuild them here. **Reason:** someone could provide a quote attachment in a message and skip all the checks we have in this file :grimacing: So my decision was to always remove quote attachments and make sure only the ones provided in `urls` are created, wdyt?

Reply:
I think that works if the URLs are sent whenever we edit a message. Once we have a test covering this decision, we are good to go.